contrib/endpoints/src/api_manager/proto/server_config.proto

```diff
@@ -143,6 +146,15 @@ message ApiAuthenticationConfig {
   bool force_disable = 1;
 }
 
+// Server config for API Authorization via Firebase Rules
+message ApiCheckSecurityRulesConfig {
+  // Firebase server to use.
+  string firebase_server = 1;
```

Comment:
We will use this configuration with t tests. I will update this in another change once I understand t tests better. For now this is only a place holder.